core/src/main/java/com/nuodb/migrator/jdbc/metadata/generator/ConnectionScriptExporter.java

```diff
@@ -55,12 +56,18 @@ protected void doOpen() throws Exception {
     }
 
     @Override
-    public void doExportScript(String script) throws Exception {
+    public void doExportScript(Script script) throws Exception {
         if (statement == null) {
             throw new GeneratorException("Connection is not opened");
         }
-        if (!StringUtils.isEmpty(script)) {
-            statement.executeUpdate(script);
+        if (!StringUtils.isEmpty(script.getSQL())) {
```

Comment:
This is the main change. Most of the rest is just fallout from changing from String to the new Script class.